### ISSUE TITLE
yeoman generator

### ISSUE THREAD

**Issue by bsorrentino** [nippur72/PolymerTS#34]
Hi

I've thought that a [yeoman](http://yeoman.io/) generator would be useful to help, this amazing project, to be widely used 

So I've released a first version of such generator available [here](https://www.npmjs.com/package/generator-polymerts)

Please test it and if you consider it useful for project's growing , add it in PolymerTS' references

**Comment by nippur72** [comment]
Sorry, I have some troubles installing it:
```
$ yo polymer:el my-element
You don't seem to have a generator with the name polymer:el installed.
You can see available generators with npm search yeoman-generator and then install them with npm install [name].
To see the 1 registered generators run yo with the `--help` option.
```
```
$ yo --help
Usage: yo GENERATOR [args] [options]

Available Generators:
    el
$
```

**Comment by bsorrentino** [comment]
Hi

I've made a mistake in documentation 

The command is :  yo polymerts:el my-element

**Comment by nippur72** [comment]
Ok! I wonder how I haven't figured it myself!

Some thoughts on the element generator (consider I'm new to yeoman):

- perhaps it's better to avoid the "app" directory and assume `yo` is launched from within the app folder?  (so it would write into `elements/` rather than `app/elements/`).

- the two link rels:
```
<link rel="import" href="..\..\bower_components/polymer/polymer.html">
<link rel="import" href="..\..\bower_components/polymer-ts.html/polymer-ts.html.html">
```
are not strictly necessary, perhaps it's better to assume that both polymer and polymer-ts have already been loaded on the main page. Not sure how this fits in a reusable elements scenario tough. 

- provide a more polished default element:

```HTML
<dom-module id="my-element">
  <template>
     <p>I'm a DOM element. This is my local DOM!</p>
     <p>And this is 'greet' property: <span>{{greet}}</span></p>
     <p><button on-click="handleClick">Change greet</button></p>
     <p>Computed property 'greetAll' is <span>{{greetAll}}</span></p>
  </template>
</dom-module>

<script type="text/javascript" src="my-element.js"></script>
```

```TypeScript
/// <reference path="../../bower_components/polymer-ts/polymer-ts.d.ts"/>

@component('my-element')
class MyElement extends polymer.Base
{
   @property({ type: String })
   greet: string = "Hello";

   @observe("greet")
   greetChanged(newValue:string, oldValue:string)
   {
      console.log(`greet has changed from ${oldValue} to ${newValue}`);
   }

   @computed()
   greetAll(test:string):string
   {
      return this.test+" to all";
   }

   // event handler 
   handleClick(e:Event)
   {
      this.greet = "Holà";      
      this.fire("greet-event");
   }
   
   @listen("greet-event")
   onButtonWasClicked(e:Event)
   {
      console.log('event "greet-event" received');                  
   }
   
   // lifecycle methods
   ready() { }
   created() { }
   attached() { }
   detached() { }
}

MyElement.register();
```

**Comment by bsorrentino** [comment]
Hi 

Thanks for feedbacks, very appreciated

About folder layout , i've chosen to follow the one proposed by [Polymer Starter Kit](https://developers.google.com/web/tools/polymer-starter-kit/) and its related [yeoman generator](https://github.com/yeoman/generator-polymer). However i'll add an option to customize output folder

I'll generate your  **polished default element** in next release

**Comment by nippur72** [comment]
Ok, thanks a lot!

**Comment by bsorrentino** [comment]
I've released the 0.2.0 including your suggestion

**Comment by nippur72** [comment]
Added to `README.md`, thanks again.
